src/components/NewGalaxyModal.js

```diff
@@ -203,8 +203,8 @@ export const NewGalaxyModal = _.flow(
   }
 
   const renderDefaultCase = () => {
-    const { cpu, memory } = _.find({ name: 'n1-standard-8' }, machineTypes)
-    const cost = getGalaxyCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8', numNodes: 2 } })
+    const { cpu, memory } = _.find({ name: 'n1-highmem-8' }, machineTypes)
+    const cost = getGalaxyCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-highmem-8', numNodes: 1 } })
```

Comment:
Might be good to factor out the default machine type so there's one place to change going forward:

```javascript
const defaultMachineType = 'n1-highmem-8'
```